scanners/ios/utility.go

```diff
@@ -640,6 +677,7 @@ func GenerateDefaultConfig(projectType XcodeProjectType) (models.BitriseConfigMa
 		true,
 		false,
 		true,
+		false,
```

Comment:
I think this way we will provide the default ios workflows for an SPM project when the manual config is selected.
This config contains steps, which are not valid for SPM and a deploy workflow.